spec/text-editor-component-spec.js

```diff
@@ -4028,9 +4028,9 @@ describe('TextEditorComponent', function () {
       component.setFontSize(10)
       await nextViewUpdatePromise()
       expect(editor.getDefaultCharWidth()).toBeCloseTo(6, 0)
-      expect(editor.getKoreanCharWidth()).toBeCloseTo(9, 0)
+      expect(editor.getKoreanCharWidth()).toBeCloseTo(10, 0)
       expect(editor.getDoubleWidthCharWidth()).toBe(10)
-      expect(editor.getHalfWidthCharWidth()).toBe(5)
+      expect(editor.getHalfWidthCharWidth()).toBeCloseTo(6, 5)
```

Comment:
Do these specs pass on Mac? If not, I think we might make this spec more general to test for differences before/after setting the font size, e.g.

```js
let originalDefaultCharWidth = editor.getDefaultCharWidth()
component.setFontSize(10)
expect(editor.getDefaultCharWidth()).toBeLessThan(originalDefaultCharWidth)
```